synapse/lib/snap.py

```diff
@@ -1621,6 +1621,24 @@ async def getNdefRefs(self, buid):
 
             yield refsbuid
 
+    async def _getLayrNdefs(self, layr, buid):
+        async for refsbuid, refsabrv in layr.getNdefRefs(buid):
+            yield refsbuid, layr.getAbrvProp(refsabrv)
+
+    # TODO: merge with getNdefRefs
```

Comment:
Should we or just keep it as a parallel API?

Reply:
Seems like merging them would be best from a maintainability perspective. Any significant drawbacks?

Reply:
Not particularly. It's one additional bool check for "do I want just the buid or both the buid and the prop". This was mostly a buy-in check in case we wanted a parallel API instead.